molecule/default/tasks/log.yml

```diff
@@ -0,0 +1,87 @@
+---
+- block:
+    - name: ensure that k8s-log namespace exists
+      k8s:
+        kind: Namespace
+        name: k8s-log
+
+    - name: create hello-world deployment
+      k8s:
+        wait: yes
+        definition:
+          apiVersion: apps/v1
+          kind: Deployment
+          metadata:
+            name: hello-world
+            namespace: k8s-log
+          spec:
+            selector:
+              matchLabels:
+                app: hello-world
+            template:
+              metadata:
+                labels:
+                  app: hello-world
+              spec:
+                containers:
+                - image: busybox
+                  name: hello-world
+                  command: ['sh']
+                  args: ['-c', 'while true ; do echo "hello world" && sleep 10 ; done']
+                restartPolicy: Always
+
+    - name: retrieve the log by providing the deployment
+      community.kubernetes.k8s_log:
```

Comment:
Technically not required since we are supplying `community.kubernetes` in the `collections` list on the playbook level; the other tests call the modules directly (without FQCN), so if we want to change to always calling the FQCN, maybe we do that in a separate issue/PR?

Reply:
just a leftover from when I was testing it separately from everything else